docker/redis/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM redis:alpine
+FROM redis:4
```

Comment:
?

Reply:
2 reasons:
- use same version in main src and fun tests
- use semver (kind of semver) instead of dodgy name. alpine can be both: v3 and v4